packages/gestalt/src/icons/ads-overview.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m2.5 19c-.828 0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5 1.5-.672 1.5-1.5-.672-1.5-1.5-1.5m19 0h-14c-.829 0-1.5.671-1.5 1.5s.671 1.5 1.5 1.5h14c.829 0 1.5-.671 1.5-1.5s-.671-1.5-1.5-1.5m-19-5c-.828 0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5 1.5-.672 1.5-1.5-.672-1.5-1.5-1.5m19 0h-14c-.829 0-1.5.671-1.5 1.5s.671 1.5 1.5 1.5h14c.829 0 1.5-.671 1.5-1.5s-.671-1.5-1.5-1.5m-11.5-3h4c.552 0 1-.448 1-1v-7c0-.552-.448-1-1-1h-4c-.552 0-1 .448-1 1v7c0 .552.448 1 1 1m12-9h-4c-.552 0-1 .448-1 1v7c0 .552.448 1 1 1h4c.552 0 1-.448 1-1v-7c0-.552-.448-1-1-1m-21 8v-7c0-.552.448-1 1-1h4c.552 0 1 .448 1 1v7c0 .552-.448 1-1 1h-4c-.552 0-1-.448-1-1" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```

Comment:
These are not optimized and compressed, running it through svgo yields a 30% reduction in size
![image](https://user-images.githubusercontent.com/7877010/57717244-a11bb980-762f-11e9-88af-2074ac0e1d04.png)

Did these come from PDS, they should be providing optimized ones